addons/website_slides/models/slide_channel.py

```diff
@@ -249,6 +249,8 @@ def _get_default_enroll_msg(self):
     can_review = fields.Boolean('Can Review', compute='_compute_action_rights', compute_sudo=False)
     can_comment = fields.Boolean('Can Comment', compute='_compute_action_rights', compute_sudo=False)
     can_vote = fields.Boolean('Can Vote', compute='_compute_action_rights', compute_sudo=False)
+    # Share URL: added because template are designed to call record.website_share_url
+    website_share_url = fields.Char('Share URL', related='website_url')
```

Comment:
Not sure, why do we need this share url on channels ? Seems we use it only for slides. Especially this feature is a new one, let us avoid unnecessary fields on channel model.

Reply:
Actually, the template "website_slides.slide_share_modal" (and some part of it directly also) is called either with a record channel or slide that's why I added also that field to Channel as well to simplify the template but indeed it is not stricly needed.
I have removed that field and passed explicitly the website_share_url while also adding a default value for security (if we forgot to pass that value) like it was done before:
```xml
<t t-if="not website_share_url">
    <t t-set="website_share_url"
       t-value="record.website_share_url if 'website_share_url' in record else record.website_url"/>
</t>
```